Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -64,6 +64,7 @@
   <ItemGroup>
     <Compile Include="AutoConfiguredNSubstituteCustomization.cs" />
     <Compile Include="AutoNSubstituteCustomization.cs" />
+    <Compile Include="AutoSubstituteCustomization.cs" />

```

Comment:
`AutoNSubstituteCustomization` has a constructor that takes an `ISpecimenBuilder` instance which is stored and returned by the `Builder` property. What is their intended use in consumer code?

With the exception of the `SubstituteAttributeRelay`, none of the new classes call NSubstitute directly. Would you leave their names without the `N`?